### ISSUE TITLE
Problem: can't create default locations

### ISSUE THREAD

**Issue by sevein** [archivematica/Issues#37]
I can't create a location (e.g. TS) via the API and make it the default location.

The existing endpoint is documented here: https://wiki.archivematica.org/Storage_Service_API#Create_new_location.

I want to add a new parameter `default` (boolean). When defined and its value is `true` I want the location to be created and be the default.

**Comment by sevein** [comment]
For QA, use the examples shared in #367 and #368, adding the `default` attribute as described in this issue, e.g.:

```
curl -s -d '{
    "pipeline": ["/api/v2/pipeline/90707555-244f-47af-8271-66496a6a965b/"],
    "purpose": "TS",
    "default": true,
    "relative_path": "automated-workflow/foo/bar",
    "description": "automated workflow",
    "space": "/api/v2/space/141593ff-2a27-44a1-9de1-917573fa0f4a/"
}' \
    -X POST \
    -H "Authorization: ApiKey test:test" \
    -H "Content-Type: application/json" \
        "http://127.0.0.1:62081/api/v2/location/" | \
            jq -r ".resource_uri"
```

**Comment by joel-simpson** [comment]
I tested creating locations of several types (backlog, transfer source, AIP Storage). I also created the actual directories and then using those locations (e.g. to start a transfer, and to store an AIP). 
I tested setting locations with default = true and default = false. 
All worked as expected. 

Note on documention - I updated the wiki with the new Default parameter. 
https://wiki.archivematica.org/Storage_Service_API#Create_new_location
I think the SS documentation on locations could be improved (the screen shot is old and doesn't show the default column).
